src/Tests/dotnet-new.Tests/Approvals/DotnetNewHelp.CanShowHelp_Install_common.verified.txt

```diff
@@ -12,4 +12,6 @@ Options:
   --interactive                                Allows the command to stop and wait for user input or action (for example to complete authentication).
   --add-source, --nuget-source <nuget-source>  Specifies a NuGet source to use during install.
   --force                                      Allows installing template packages from the specified sources even if they would override a template package from another source.
+  -v, --verbosity <LEVEL>                      Sets the verbosity level. Allowed values are q[uiet], m[inimal], n[ormal], and diag[nostic]. [default: normal]
+  -d, --diagnostics                            Enable diagnostic output. [default: False]
```

Comment:
Some option description uses infinitive and some are using 3rd form. What is a guideline here?

Reply:
Align them with the 3rd person for `new` to stay aligned with all existing commands if possible. We don't have guidelines for this for the entire CLI right now, so it's more important to stay consistent within the command.

Reply:
Adjusted everything to use 3rd person.

The only exception is `--help`:  there is no easy way to change that and it remains `Show command line help.`